admiral/envs/components/agent.py

```diff
@@ -219,29 +232,48 @@ class SpeedAngleAgent(Agent):
     initial_ground_angle (float):
         The agent's initial ground angle.
     """
-    def __init__(self, min_speed=0.25, max_speed=1.0, max_acceleration=0.25, \
-                 max_banking_angle=45, max_banking_angle_change=30, \
-                 initial_speed=None, initial_banking_angle=None, \
-                 initial_ground_angle=None, **kwargs):
+    def __init__(self, min_speed=0.25, max_speed=1.0, max_banking_angle=45, initial_speed=None, \
+                 initial_banking_angle=None, initial_ground_angle=None, **kwargs):
         super().__init__(**kwargs)
         self.min_speed = min_speed
         self.max_speed = max_speed
-        self.max_acceleration = max_acceleration
         self.initial_speed = initial_speed
         self.speed = None # Should be set by the state handler
 
         self.max_banking_angle = max_banking_angle
-        self.max_banking_angle_change = max_banking_angle_change
         self.initial_banking_angle = initial_banking_angle
         self.initial_ground_angle = initial_ground_angle
         self.banking_angle = None # Should be set by the state handler
     
     @property
     def configured(self):
         return super().configured and self.min_speed is not None and self.max_speed is not None and \
-            self.max_acceleration is not None and self.max_banking_angle is not None and \
+            self.max_banking_angle is not None
+
+class SpeedAngleActingAgent(ActingAgent):
+    """
+    Agents can change their speed and banking angles.
+    
+    max_acceleration (float):
```

Comment:
This can be broken into Speed and Angle agents. Then we can merge the accelerating part with AcceleratingAgent below and the actors and states can determine how to use that information (velocity updater vs speed-angle updater).

Reply:
#53